source/comply/data-retention-policy.rst

```diff
@@ -61,12 +80,6 @@ This usually means another data retention job is in progress. You can verify thi
 
 If no jobs are in progress and the job has stayed ``Pending`` for more than 2 minutes, then you may not have restarted your server after enabling the data retention policy. Restart your server and try again.
```

Comment:
Is a server restart required when making global or custom data retention policies?

Reply:
@mkraft Can you confirm if a restart is still required for making changes to the data retention policies?

Reply:
No it's not necessary anymore.

Reply:
Great Thanks!!